run_bot.py

```
@@ -0,0 +1,18 @@
+# run_bot.py
+
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)